website/source/api/index.html.md

```diff
@@ -77,6 +77,47 @@ to the supplied maximum `wait` time to spread out the wake up time of any
 concurrent requests. This adds up to `wait / 16` additional time to the maximum
 duration.
 
+### Implementation Details
+
+While the mechanim is relatively simple to work with, there are a few subtelties
```

Comment:
```suggestion
While the mechanism is relatively simple to work with, there are a few subtleties
```

Reply:
Can we replace subtleties with a more commonly used phrase? Perhaps, "characteristics" or "qualities".

Reply:
Maybe guidelines?

Reply:
I guess the intent is that these are non-obvious ways that the naive implementation can fail. "Subtelties" implies that.

I'm not sure any of those other suggestions are quite applicable. Do you think it would be clearer to say: "...there are a few subtle edge cases that must be handled correctly"?

Reply:
Yes, I do like that phrasing better.